Product/configuration-management.md

```diff
@@ -406,24 +408,31 @@ Example of the typology configuration metadata:
 
 ### The rules object
 
-The `rules` object is an array that contains an element for every possible outcome for each of the rule results that can be received from the rule processors in scope for the typology.
+The `rules` object is an array that contains each rule in scope for the typology, and within each rule there is an array for every possible outcome for the rule results that can be received from the rule processors
 
-![Tazama rule processor](../images/tazama-rule-and-typology-processor-detail.drawio.svg)
-
-***Every. Possible. Outcome.***
-
-All the possible outcomes from the rule processors are encapsulated in each rule's configuration, with the exception of the `.err` outcome that is not listed in the rule configuration because the conditions and descriptions are built into the rule processor itself. When composing the typology configuration, the user must remember to include the `.err` outcome, but the rest of the rule results (exit conditions and banded/cased results) can be directly reconciled with the elements in the `rules` object.
-
-Each rule result element in the rules array contains the same attributes:
+Each rule result element in the rules array contains the attributes:
 
 | **Attribute** | **Description** |
 | --- | --- |
 | `id` | The rule processor that was used to determine the rule result is uniquely identified by this identifier attribute. |
 | `cfg` | The configuration version attribute specifies the unique version of the rule configuration that was used by the processor to determine this result. |
+| `termId` | The unique identifier for the rule outcome. |
```

Comment:
This may need some additional explanation for how this variable relates to the typology expression somewhere.

Reply:
Updated